image/cli/mascli/functions/gitops_db2u_database

```diff
@@ -268,6 +284,7 @@ function gitops_db2u_database_noninteractive() {
   [[ -z "$DB2_DATA_STORAGE_CLASS" ]] && gitops_db2u_help "DB2_DATA_STORAGE_CLASS is not set"
   [[ -z "$DB2_BACKUP_STORAGE_CLASS" ]] && gitops_db2u_help "DB2_BACKUP_STORAGE_CLASS is not set"
   [[ -z "$DB2_LOGS_STORAGE_CLASS" ]] && gitops_db2u_help "DB2_LOGS_STORAGE_CLASS is not set"
+  [[ -z "$DB2_AUDIT_LOGS_STORAGE_CLASS" ]] && gitops_db2u_help "DB2_AUDIT_LOGS_STORAGE_CLASS is not set"
```

Comment:
The audit logs are optional so we shouldn't have DB2_AUDIT_LOGS_STORAGE_CLASS as a requirement